x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -53,7 +54,7 @@ export const setSignalsStatusRoute = (
         },
       },
       async (context, request, response) => {
-        const { status } = request.body;
```

Comment:
reason field should be defined in API schema: x-pack/solutions/security/plugins/security_solution/common/api/detection_engine/signals/set_signal_status/set_signals_status_route.schema.yaml

Reply:
Should it also be validated? And schema should not allow random string to be submitted?